.github/workflows/test-deploy.yml

```diff
@@ -69,11 +69,21 @@ jobs:
           cml runner --version
           cml publish --version
           cml pr --version
+
+          npm install
+          pkg -t node14-linux,node14-macos package.json
```

Comment:
I did not remember that we now we support node16 instead node14, in any case only one is needed since its going to be wrapped into an executable file. Does not really matter indeed which one while the code works

Reply:
Related to the arch I think we are fine.